spec/api/v1/api_crops_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Api::V1::CropsController do
+
+  before(:each) do
+    @crop = FactoryGirl.create(:tomato)
+    @crop.save
+  end
+
+  describe "GET crops index" do
+    it 'fetches the crops index' do
+      get '/api/v1/crops'
+      response.should be_success
+      response.content_type.should eq("application/json")
+    end
```

Comment:
Also, what do you guys think of the be_success matcher? I generally choose the more specific numeric codes and that seems like it might be better when testing an API.